backend/src/v5/schemas/tickets/validators.js

```diff
@@ -72,13 +72,18 @@ const generateViewValidator = (isUpdate, required) => {
 		}),
 	).default(undefined), true);
 
-	const validator = Yup.object().shape({
-		screenshot: types.embeddedImage(isUpdate),
+	let validator = Yup.object({
 		state,
 		camera: !isUpdate && required ? camera.required() : camera,
 		clippingPlanes,
 	}).default(undefined);
 
+	if (!isComment) {
```

Comment:
Think it's clearer if we do this before we turn it into a yup object

i.e.
```suggestion
	const schema = {
	   //...
	   state,
	   camera: !isUpdate && required ? camera.required() : camera,
	   clippingPlanes,
	}
	
	if(!isComment) {
	   schema.screenshot = types.embeddedImage(isUpdate)
	}
	
	return imposeNullableRule(Yup.object(schema).default(undefined));
```